JavaAssignments/src/shraddha/LargeSmallNum.java

```diff
@@ -0,0 +1,58 @@
+//---------------------------------------------------------------------------------------------------
+//Largest and smallest number in unsorted array? 
+//Hint: For example, if input integer array is {2, 6, 3, 9, 11} and given sum is 9, output should be {6,3}. 
+//---------------------------------------------------------------------------------------------------
+
+package shraddha;
+
+import java.util.Scanner;
+
+public class LargeSmallNum {
+
+	Scanner sc = new Scanner(System.in);
+
+	// Accepting the number series
+	int[] acceptNumberSeries() {
+		System.out.println("Enter total number Range: ");
+		int numberRange = sc.nextInt();
+		int[] number_series = new int[numberRange];
+
+		System.out.println("Please enter the " + numberRange + " numbers in a series: ");
+		for (int index = 0; index < number_series.length; index++) {
+			number_series[index] = sc.nextInt();
+		}
+		return number_series;
+	}
+
+	void largeSmallNum(int[] number_series) {
+		System.out.println("Enter sum output/number : ");
+		int number = sc.nextInt();
+		int min = 0, max = 0;
+		for (int num1 = 0; num1 < number_series.length - 1; num1++) {
```

Comment:
We just need to find largest and smallest number. why we  should write code in n*n complexity.
There should be twp simple method findLargest, findSmallest

Reply:
need to talk about this logic.